datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -909,4 +911,12 @@ private void configureGlossaryRelationshipResolvers(final RuntimeWiring.Builder
             }
         }), loaderOptions);
     }
+    private void configureGlossaryTermResolvers(final RuntimeWiring.Builder builder) {
+        builder
+                .type("GlossaryTerm", typeWiring -> typeWiring
+                        .dataFetcher("schemaMetadata", new AuthenticatedResolver<>(
+                                new AspectResolver())
+                        )
+                );
+    }
```

Comment:
lets put this method near the other configureXyzResolvers

Reply:
done